src/schema/objects/columns.yaml

```diff
@@ -0,0 +1,507 @@
+    (not needed when it is common to all channels, in that case it can be specified in `*_eeg.json` as `EEGReference`).
+  type: string
+respiratory:
+  name: respiratory
+  description: |
+    continuous breathing measurement
+  type: number
+response_time:
+  name: response_time
+  description: |
+    Response time measured in seconds.
+    A negative response time can be used to represent preemptive responses and
+    "n/a" denotes a missed response.
+  anyOf:
+  - type: number
+    unit: s
+  - type: string
+    enum:
+    - n/a
+sample:
+  name: sample
+  description: |
+    Onset of the event according to the sampling scheme of the recorded modality
+    (that is, referring to the raw data file that the `events.tsv` file accompanies).
+  type: integer
+sample_id:
+  name: sample_id
+  description: |
+    MUST consist of `sample-<label>` values identifying one row for each sample.
+  type: string
+  pattern: ^sample-[0-9a-zA-Z]+$
+sample_type:
+  name: sample_type
+  description: |
+    MUST consist of sample type values, either `cell line`, `in vitro differentiated cells`,
+    `primary cell`, `cell-free sample`, `cloning host`, `tissue`, `whole organisms`,
+    `organoid`, or `technical sample` from
+    [ENCODE Biosample Type](https://www.encodeproject.org/profiles/biosample_type).
+  type: string
+  enum:
+  - cell line
+  - in vitro differentiated cells
+  - primary cell
+  - cell-free sample
+  - cloning host
+  - tissue
+  - whole organisms
+  - organoid
+  - technical sample
+sampling_frequency:
+  name: sampling_frequency
+  description: |
+    Sampling rate of the channel in Hz.
+  type: number
+  unit: Hz
+session_id:
+  name: session_id
+  description: |
+    MUST consist of `ses-<label>` values identifying one row for each session.
+  type: string
+  pattern: ^ses-[0-9a-zA-Z]+$
+sex:
+  name: sex
+  description: |
+    String value indicating phenotypical sex, one of "male", "female", "other".
+
+    For "male", use one of these values: `male`, `m`, `M`, `MALE`, `Male`.
+
+    For "female", use one of these values: `female`, `f`, `F`, `FEMALE`, `Female`.
+
+    For "other", use one of these values: `other`, `o`, `O`, `OTHER`, `Other`.
+  type: string
+  enum:
+  - male
+  - m
+  - M
+  - MALE
+  - Male
+  - female
+  - f
+  - F
+  - FEMALE
+  - Female
+  - other
+  - o
+  - O
+  - OTHER
+  - Other
+  - n/a
+size:
+  name: size
+  description: |
+    Surface area of the electrode, units MUST be in `mm^2`.
+  type: number
+  unit: mm2
+software_filters:
+  name: software_filters
+  description: |
+    List of temporal and/or spatial software filters applied
+    (for example, "SSS", `"SpatialCompensation"`).
+    Note that parameters should be defined in the general MEG sidecar .json file.
+    Indicate `n/a` in the absence of software filters applied.
+  anyOf:
+  - type: string
+  - type: string
+    enum:
+    - n/a
+status:
+  name: status
+  description: |
+    Data quality observed on the channel `(good/bad)`.
```

Comment:
I think we should also clarify that `n/a` is a valid entry next to good and bad.